index.js

```diff
@@ -58,7 +58,17 @@ var handlers = {
         this.emit('AMAZON.StopIntent');
     },
     'AMAZON.HelpIntent': function () {
-        this.emit('Unhandled');
+        var self = this;
+        ApiAi.eventRequest({name: 'HELP'}, {sessionId: alexaSessionId})
+            .on('response', function (response) {
+                var speech = response.result.fulfillment.speech;
+                self.emit(':ask', speech, speech);
```

Comment:
Note that _:ask_ keeps the session open when a user requests help. This is the behavior amazon expects.